Common/UnitDefinitions/Frequency.json

```diff
@@ -67,6 +67,18 @@
           "Abbreviations": [ "bpm" ]
         }
       ]
+    },
+    {
+      "SingularName": "PerSecond",
+      "PluralName": "PerSecond",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "s⁻¹" ]
```

Comment:
Encoding error? Doesn't look right in Chrome on Android at least.

Reply:
I saw the same thing on Andriod, but looks fine on Chrome on Windows. I intentionally saved this file as UTF-8 without BOM. Let me know if it should be a different encoding

Reply:
Looks good here too